docs/pages/organization.rst

```diff
@@ -0,0 +1,5 @@
+Repository / package organization
```

Comment:
I think a link is missing somewhere- I can't find this page in the generated docs